src/Scene/Description/Style.js

```diff
@@ -1,154 +0,0 @@
-/**
```

Comment:
> Yes this code is going to be used soon

Then whoever wants them just need to run
```bash
git checkout acea086 -- src/Scene/Description
```

That will bring them back in the tree. In the meantime, let's not put the burden to maintain dead code on *all* contributors (I have already 2 commits in master that touches these files).